src/metal.cs

```diff
@@ -3316,8 +3800,13 @@ interface IMTLIndirectRenderCommand {}
 	[Mac (10,14), iOS (12,0)]
 	[Protocol]
 	interface MTLIndirectRenderCommand {
-		[NoiOS, NoTV, Mac (10,14)]
+#if XAMCORE_4_0
+		[Abstract]
```

Comment:
This doesn't look quite right: you're adding a `[Abstract]` conditional on XAMCORE_4_0, but there's already an `[Abstract]` on this member.